compiler/env/OMRCompilerEnv.hpp

```diff
@@ -99,6 +100,10 @@ class OMR_EXTENSIBLE CompilerEnv
    //
    TR::DebugEnv debug;
 
+   // Reference to the Ahead of Time compiler infrastructure in OMR
+   //
+   TR::AOTLoadStoreDriver* aotLoadStoreDriver;
```

Comment:
small tip: better to associate the `*` with the variable name and not the type because that's how it is actually parsed (so recommend you write `TR::AOTLoadStoreDriver *aotLoadStoreDriver`). I know there's only one variable there, but it's a good mental habit.

Reply:
Thank you, I will try to develop this habit and definitely implement the change!